src/components/regions/sectorChart/RegionalSectorPieChart.tsx

```diff
@@ -0,0 +1,59 @@
+import { PieChart } from "@/components/charts/PieChart";
```

Comment:
there are some issues with the imports here, but since this is not used anywhere yet and is just prep for later work, didn't review this too closely for the moment